.env.example

```diff
@@ -4,3 +4,7 @@ RABBITMQ_USERNAME=guest
 RABBITMQ_PASSWORD=guest
 HEARTBEAT_INTERVAL_SEC=5
 LOG_LEVEL=debug
+# Set to true to disable database connection
+SQLX_OFFLINE=false
```

Comment:
I don't get it. When would we want to disable db connection?

Reply:
for local or ci workflows
simple scenario, for any given reason you have empty db, app will not build to run migrations. you can remove `DATABASE_URL` from env or bypass with `SQLX_OFFLINE`
this is actually what Łukasz has run into and I believe that an example with desc could have been beneficial.
however I will comment this more appropriately